### PR TITLE
Bump to 9.4.0

### PR DESCRIPTION
**Pull request by scpeters**
Update Changelog and Migration guide.

https://github.com/osrf/sdformat/compare/sdformat9_9.3.0...scpeters:bump94

### CODE REVIEW

**Review by nkoenig — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -1,8 +1,39 @@
...
+    * [Pull request 399](https://github.com/osrf/sdformat/pull/399)
+
+1. Add Sky DOM
+    * [Pull request 417](https://github.com/osrf/sdformat/pull/417)
+
+1. camera.sdf: decrease far clip lower bound
+    * [Pull request 435](https://github.com/osrf/sdformat/pull/435)
+
+1. material.sdf: add `double_sided` parameter
+    * [Pull request 435](https://github.com/osrf/sdformat/pull/435)
```
Comment (azeey) [inline Changelog.md:34]: Is this #410?

**Review by azeey — APPROVED** [review]

**Review by chapulina — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (scpeters) [reply Changelog.md]: nice catch; fixed in 5c211e50c709cd073c4b2a25fd9f05f64b5587da

**Comment by chapulina** [comment]
Can we update this to include the latest additions and trigger a release?

https://github.com/osrf/sdformat/compare/sdformat9_9.3.0...sdf9

We may also be able to get #454 in

**Comment by chapulina** [comment]
This has lots of approvals, I'm merging and triggering the release.

**Pull request merged**